doc/classes/Color.xml

```diff
@@ -403,26 +415,26 @@
 			<return type="String" />
 			<param index="0" name="with_alpha" type="bool" default="true" />
 			<description>
-				Returns the color converted to an HTML hexadecimal color string in RGBA format (ex: [code]ff34f822[/code]).
-				Setting [param with_alpha] to [code]false[/code] excludes alpha from the hexadecimal string (and uses RGB instead of RGBA format).
+				Returns the color converted to an HTML hexadecimal color [String] in RGBA format, without the hash ([code]#[/code]) prefix.
+				Setting [param with_alpha] to [code]false[/code], excludes alpha from the hexadecimal string, using RGB format instead of RGBA format.
 				[codeblocks]
 				[gdscript]
-				var color = Color(1, 1, 1, 0.5)
-				var with_alpha = color.to_html() # Returns "ffffff7f"
-				var without_alpha = color.to_html(false) # Returns "ffffff"
+				var white = Color(1, 1, 1, 0.5)
+				var with_alpha = white.to_html() # Returns "ffffff7f"
+				var without_alpha = white.to_html(false) # Returns "ffffff"
 				[/gdscript]
 				[csharp]
-				var color = new Color(1, 1, 1, 0.5f);
-				String withAlpha = color.ToHtml(); // Returns "ffffff7f"
-				String withoutAlpha = color.ToHtml(false); // Returns "ffffff"
+				var white = new Color(1, 1, 1, 0.5f);
+				String withAlpha = white.ToHtml(); // Returns "ffffff7f"
+				String withoutAlpha = white.ToHtml(false); // Returns "ffffff"
```

Comment:
In C# we should prefer the keyword `string` to the framework type name.

```suggestion
				string withAlpha = white.ToHtml(); // Returns "ffffff7f"
				string withoutAlpha = white.ToHtml(false); // Returns "ffffff"
```

Usually for boolean parameters it's common to include the parameter name because otherwise it's not clear what it is, but it's not a strict rule and I'm not sure if we do it anywhere else in the documentation. It would look like this:

```csharp
white.ToHtml(includeAlpha: false);
```

Reply:
> Usually for boolean parameters it's common to include the parameter name because otherwise it's not clear what it is, but it's not a strict rule and I'm not sure if we do it anywhere else in the documentation. It would look like this:
> 
> ```cs
> white.ToHtml(includeAlpha: false);
> ```

I didn't even know it was a possibility! I mean, I would certainly do that, but I've not ever seen it, nor there's guidelines for it, but it is definitely worth noting in the future. One could discuss it with other maintainers and go through the class reference to retroactively add it everywhere it matters.